test/lib/interpreters/split_clustering_functions_test.rb

```diff
@@ -0,0 +1,86 @@
+# Copyright © Mapotempo, 2021
```

Comment:
I don't think this PR  is the right place for needs this new file and moving `test_if_duration_from_and_to_depot_is_filled_correctly`, as we have an on-going discussion in another PR exactly for this purpose.

I think we should continue with the current rule until we finally decide what to do and after then we can do such moves in a PR dedicated to this task.